pubspec.yaml

```diff
@@ -1,15 +1,16 @@
 name: logging
-version: 0.11.3+2
+version: 0.11.4-dev
 
-description: >
+description: >-
   Provides APIs for debugging and error logging. This library introduces
   abstractions similar to those used in other languages, such as the Closure
   JS Logger and java.util.logging.Logger.
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/logging
 
 environment:
-  sdk: '>=1.5.0 <3.0.0'
+  sdk: '>=2.0.0 <3.0.0'
 
 dev_dependencies:
-  test: '>=0.12.0 <2.0.0'
+  pedantic: ^1.3.0
```

Comment:
It seems that in general folks are pinning versions when pulling in `pedantic`.  For example:

https://github.com/dart-lang/test/blob/957a0f8fbd3dc21de746d448ff313c148b3e51c7/pkgs/test/pubspec.yaml#L48

Are you doing differently elsewhere?

Reply:
Our thought is that we should not be pinning – so we pick up the latest as things evolve. If that means travis breaks and we have to fix – it's okay.